submissions/YuraZagor/tjsw/index.js

```diff
@@ -0,0 +1,154 @@
+const manNamesArr = ['Vasil', 'Yurij', 'Zahar', 'Taras', 'Olexiy', 'Oleg', 'Mykola', 'Sashko', 'Sergij', 'Bohdan']
+const womanNamesArr = ['Hanna', 'Helena', 'Vasilisa', 'Yulya', 'Oksana', 'Olena', 'Olga', 'Myroslava', 'Svitlana', 'Bohdana']
+const animalNamesArr = ['Joy', 'Happy', 'Lucky', 'Puffy']
+
+let catSay = 'meau';
+let dogSay = 'woufff';
+let humanSay = 'Glory to Ukraine!';
+
+function randomSelect(arr){
+  return arr[Math.floor(Math.random()*arr.length)]
+}
+let manName = randomSelect(manNamesArr)
+let womanName = randomSelect(womanNamesArr)
+let animalName = randomSelect(animalNamesArr)
+
+const inhabitants = [
+  { species: 'human', 
+    gender: 'male',
+    name: 'Vasil',
+    saying: 'Slava natsiyi',
+    friend: 'Hanna',
+    legs: 2,
+    hands: 2,
+  },
+  { species: 'human', 
+    gender: 'female',
+    name: 'Hanna',
+    saying: 'Smert vorogam',
+    friend: 'Vasil',
+    legs: 2,
+    hands: 2,
+  },
+];
+
+class Inhabitant {  
+  constructor(species, name, saying, friend, gender, habitat=inhabitants){
```

Comment:
If neither child class passes a value for `habitat` why we introduce this parameter at all?
Also class shouldn't be aware of any global variables.
The requirements you learned from Frogger task are still applicable.

Reply:
I see.
Added a habitat for dogs only - dogsVolier, so now this param makes more sence